dbms/tests/queries/0_stateless/00917_least_sqr.sql

```diff
@@ -0,0 +1,7 @@
+select arrayReduce('leastSqr', [1, 2, 3, 4], [100, 110, 120, 130]);
```

Comment:
Input format is not quite clear from my point of view.
First array contains X, and second - Y ?

Reply:
Yes. A more clear scenario is `select leastSqr(x, y) from points;`